### PR TITLE
Nuzhy/85535/refactor our principle page

### PR DESCRIPTION
**Pull request by Nuzhy-Deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/our-principles/index.tsx [hunk 1]
```diff
@@ -118,14 +116,16 @@ const OurPrinciples = () => {
                             </StyledFlex>
                             <Flex fd="column" max_width="792px">
                                 <ColorHeader as="h2" color={principle.color}>
-                                    {principle.title}
+                                    <Localize translate_text={principle.title} />
                                 </ColorHeader>
                                 {principle.description.map((description, key) => (
                                     <Flex key={key} fd="column">
```
Comment (NikitK-deriv) [inline src/pages/our-principles/index.tsx:122]: Nitpick: I realize that isn't a part of task but we can improve it a little bit

```suggestion
                                {principle.description.map((description) => (
                                    <Flex key={description.title} fd="column">
```

**Review by NikitK-deriv — APPROVED** [review]

File: src/pages/our-principles/index.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { principles } from './_data'
 import Layout from 'components/layout/layout'
 import { Header, Text } from 'components/elements'
```
Comment (aswathy-deriv) [inline src/pages/our-principles/index.tsx:5]: You can remove the Text and replace it with Header..Text is deprecated

File: src/pages/our-principles/index.tsx (hunk 1 above)
Comment (Nuzhy-Deriv) [reply src/pages/our-principles/index.tsx]: I have changed as advised.

**Review by niloofar-deriv — APPROVED** [review]
LGTM :+1:

**Pull request closed without merging**